src/test/resources/sql/insert_cases_get_case_links.sql

```diff
@@ -3,7 +3,7 @@ DELETE FROM case_data;
 DELETE FROM case_link;
 
 
-INSERT INTO case_data (id, case_type_id, jurisdiction, state, security_classification, data, reference)
+INSERT INTO case_data (id, case_type_id, jurisdiction, state, security_classification, data, data_classification, reference, created_date, last_modified, last_state_modified_date)
```

Comment:
I removed `data_classification, created_date, last_modified, last_state_modified_date` as they are not mandatory and I was finding the sql file was getting confusingly large and long winded.

Reply:
Had to reinsert `data_classifciation` to stop data being filtered out during integration tests. and `created_date` allowed me some confidence in sort order.  However I possibly could/should have skipped `last_modified, last_state_modified_date`.